Gems/OpenXRVk/Code/CMakeLists.txt

```diff
@@ -79,9 +79,13 @@ ly_add_target(
             Gem::OpenXRVk.Static
 )
 
-# use the OpenXRVk module in clients and tools:
+# use the OpenXRVk module in all aliases:
 ly_create_alias(NAME OpenXRVk.Clients NAMESPACE Gem TARGETS Gem::OpenXRVk)
-ly_create_alias(NAME OpenXRVk.Tools NAMESPACE Gem TARGETS Gem::OpenXRVk)
+
+if(PAL_TRAIT_BUILD_HOST_TOOLS)
+    ly_create_alias(NAME OpenXRVk.Tools NAMESPACE Gem TARGETS Gem::OpenXRVk)
+    ly_create_alias(NAME OpenXRVk.Builders NAMESPACE Gem TARGETS Gem::OpenXRVk)
```

Comment:
You may want to update the gem to reference the `${gem_name}` variable to make it easier to rename in the future.
https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/CMakeLists.txt
https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/Code/CMakeLists.txt

Reply:
The "parent" cmake file doesn't appear to have set the `${gem_name}` variable:
https://github.com/o3de/o3de-extras/blob/development/Gems/OpenXRVk/CMakeLists.txt

Which makes me think this gem wasn't instantiated from the template.

Reply:
Probably not